analysis/src/main.rs

```diff
@@ -23,57 +16,52 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .open()
         .unwrap();
 
-    let mut ethernet_frame: ethernet_frame::EthernetFrame = Default::default();
-    let mut ipv4: ip_protocol::IPv4 = Default::default();
-    let _ipv6: ip_protocol::IPv6 = Default::default();
-    let mut counter: u16 = 1;
-    ctrlc::set_handler(|| {
-        KILL_EXECUTION.store(true, std::sync::atomic::Ordering::Relaxed);
-    })?;
+    for _ in 0.. {
+        if let Ok(packet) = cap.next_packet() {
```

Comment:
Skift til at bruge `while let` i stedet for `for _ in 0..`

Reply:
Og so heller quit med programmet med `ctrlc::set_handler`

Reply:
Fixed loop. Ctrl-c virker fint på min maskine :)